docs/pubsub_snippets.py

```diff
@@ -66,6 +66,28 @@ def do_something_with(sub):  # pylint: disable=unused-argument
 
 
 @snippet
+def client_topic(client, to_delete):  # pylint: disable=unused-argument
+    """Topic factory."""
+    TOPIC_NAME = 'topic_factory-%d' % (_millis(),)
+
+    # [START client_topic]
+    topic = client.topic(TOPIC_NAME)
+    # [END client_topic]
+
+
+@snippet
+def client_subscription(client, to_delete):  # pylint: disable=unused-argument
+    """Subscription factory."""
+    SUBSCRIPTION_NAME = 'subscription_factory-%d' % (_millis(),)
+
+    # [START client_subscription]
+    subscription = client.topic(
```

Comment:
I'll follow up with a PR addressing the missing `topic` in `Subscription.reload()`